homeassistant_cli/remote.py

```diff
@@ -89,13 +89,13 @@ def restapi(
 
 def wsapi(
     ctx: Configuration, frame: Dict, wait: bool = False
-) -> Optional[Dict]:
-    """Make a call to Home Assistant using WS API."""
-    async def fetcher() -> Optional[Dict]:
+) -> Optional[str]:
```

Comment:
this wouldn't be the right long term fix as code calling wsapi like restapi would expect a structured return.

Reply:
Yeah, it doesn't make sense to dump the content to string if it's a single response.